WebDevStudios/wds-bt
# Reorganize scss files

# Closes

N/A

## Link to test

N/A

## Description

* Reorganize scss files and directories
  * Make directories more literal (e.g. `template-parts` to style WordPress Template Parts)
  * Use `_index.scss` so we can get rid of the double import (`@import './base'` instead of `@import './base/base'`)
  * Remove underscore from scss files other than index, so that index can be at the top of the dir
    * The underscores are not required anyway, this keeps things nice and organized visually
  * Remove the `components` directory and move form styles to `base`
    * In the spirit of literal directory names, since there is nothing called "components" in WP, this PR removes this dir and moves the form styles to `base` since they style bare html elements like `input` and `select`

## Mobile & Desktop Screenshots/Recordings

![2024-06-21-11 50 20](https://github.com/WebDevStudios/wds-bt/assets/12365909/5d43a59f-da2e-4785-a522-514161f5210b)

![2024-06-21-11 51 04](https://github.com/WebDevStudios/wds-bt/assets/12365909/a8034fd2-cbf8-42f1-abfa-047719942252)

## Added to documentation?


## Testing Instructions

1. Pull this branch, build the theme
2. Verify that styles are building correctly

-----

## Reviewer's Testing Checklist



As a reviewer, please verify that the relevant testing criteria are fulfilled and confirmed before approving this Pull Request.


## [optional] Additional Reviewer Notes or Considerations?
